File: cite_culture/src/Entities/Cours.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author Farah
 */
public class Cours 
{
    private int id_cours;
    private String type;
    private int id_prof;
    private int nb_places;
    private String image;
    private int id_salle;
    private Date date;
    private String heure_debut;
    private String heure_fin;
    private float prix;

    
    public Cours(){
        
    }
    
     public Cours(String type, int id_prof, int nb_places, String image, int id_salle, Date date, String heure_debut, String heure_fin, float prix) {
       // this.id_cours = id_cours;
        this.type = type;
        this.id_prof = id_prof;
        this.nb_places = nb_places;
        this.image = image;
        this.id_salle = id_salle;
        this.date = date;
        this.heure_debut = heure_debut;
        this.heure_fin = heure_fin;
        this.prix = prix;
    }

    public int getId_cours() {
        return id_cours;
    }

    public String getType() {
        return type;
    }

    public int getId_prof() {
        return id_prof;
    }

    public int getNb_places() {
        return nb_places;
    }

   

    public int getId_salle() {
        return id_salle;
    }

    public Date getDate() {
        return date;
    }

   

    public String getHeure_debut() {
        return heure_debut;
    }

    public String getHeure_fin() {
        return heure_fin;
    }

    public float getPrix() {
        return prix;
    }

    public String getImage() {
        return image;
    }
    

    public void setId_cours(int id_cours) {
        this.id_cours = id_cours;
    }

    public void setType(String type) {
        this.type = type;
    }

    public void setId_prof(int id_prof) {
        this.id_prof = id_prof;
    }

    public void setNb_places(int nb_places) {
        this.nb_places = nb_places;
    }

    
    

    public void setId_salle(int id_salle) {
        this.id_salle = id_salle;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public void setHeure_debut(String heure_debut) {
        this.heure_debut = heure_debut;
    }

    public void setHeure_fin(String heure_fin) {
        this.heure_fin = heure_fin;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }

    public void setImage(String image) {
        this.image = image;
    }

    @Override
    public String toString() {
        return "Cours{" + "id_cours=" + id_cours + ", type=" + type + ", id_prof=" + id_prof + ", nb_places=" + nb_places + ", image=" + image + ", id_salle=" + id_salle + ", date=" + date + ", heure_debut=" + heure_debut + ", heure_fin=" + heure_fin + ", prix=" + prix + '}';
    }
    
    
}

File: src/AppBundle/Entity/Cours.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Cours
 *
 * @ORM\Table(name="cours", indexes={@ORM\Index(name="id_prof", columns={"id_prof"}), @ORM\Index(name="id_salle", columns={"id_salle"})})
 * @ORM\Entity
 */
class Cours
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_cours", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idCours;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=200, nullable=false)
     */
    private $type;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_prof", type="integer", nullable=false)
     */
    private $idProf;

    /**
     * @var integer
     *
     * @ORM\Column(name="nb_places", type="integer", nullable=false)
     */
    private $nbPlaces;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_salle", type="integer", nullable=false)
     */
    private $idSalle;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var string
     *
     * @ORM\Column(name="heureDebut", type="string", length=200, nullable=false)
     */
    private $heuredebut;

    /**
     * @var string
     *
     * @ORM\Column(name="heureFin", type="string", length=200, nullable=false)
     */
    private $heurefin;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=200, nullable=false)
     */
    private $image;



    /**
     * Get idCours
     *
     * @return integer
     */
    public function getIdCours()
    {
        return $this->idCours;
    }

    /**
     * Set type
     *
     * @param string $type
     *
     * @return Cours
     */
    public function setType($type)
    {
        $this->type = $type;

        return $this;
    }

    /**
     * Get type
     *
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * Set idProf
     *
     * @param integer $idProf
     *
     * @return Cours
     */
    public function setIdProf($idProf)
    {
        $this->idProf = $idProf;

        return $this;
    }

    /**
     * Get idProf
     *
     * @return integer
     */
    public function getIdProf()
    {
        return $this->idProf;
    }

    /**
     * Set nbPlaces
     *
     * @param integer $nbPlaces
     *
     * @return Cours
     */
    public function setNbPlaces($nbPlaces)
    {
        $this->nbPlaces = $nbPlaces;

        return $this;
    }

    /**
     * Get nbPlaces
     *
     * @return integer
     */
    public function getNbPlaces()
    {
        return $this->nbPlaces;
    }

    /**
     * Set idSalle
     *
     * @param integer $idSalle
     *
     * @return Cours
     */
    public function setIdSalle($idSalle)
    {
        $this->idSalle = $idSalle;

        return $this;
    }

    /**
     * Get idSalle
     *
     * @return integer
     */
    public function getIdSalle()
    {
        return $this->idSalle;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return Cours
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set heuredebut
     *
     * @param string $heuredebut
     *
     * @return Cours
     */
    public function setHeuredebut($heuredebut)
    {
        $this->heuredebut = $heuredebut;

        return $this;
    }

    /**
     * Get heuredebut
     *
     * @return string
     */
    public function getHeuredebut()
    {
        return $this->heuredebut;
    }

    /**
     * Set heurefin
     *
     * @param string $heurefin
     *
     * @return Cours
     */
    public function setHeurefin($heurefin)
    {
        $this->heurefin = $heurefin;

        return $this;
    }

    /**
     * Get heurefin
     *
     * @return string
     */
    public function getHeurefin()
    {
        return $this->heurefin;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Cours
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return Cours
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }
}
